P/polyfill_glibc/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "polyfill_glibc"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/corsix/polyfill-glibc.git", "f2005b78d6d0402ddc1079bcfb6db31a424f663b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/polyfill-glibc/
+ninja -j${nproc} polyfill-glibc
+install -D ./polyfill-glibc ${prefix}/bin/polyfill-glibc
```

Comment:
```suggestion
install -Dvm 755 ./polyfill-glibc ${prefix}/bin/polyfill-glibc
```
Just a nit, we like to be explicit with the permissions when installing the binaries.

Reply:
Added.